mne_bids/update.py

```diff
@@ -0,0 +1,96 @@
+"""Update BIDS compatible directory structures and meta data."""
+# Authors: Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
+
+import json
```

Comment:
The module name and the comment are too generic…

Reply:
I've updated it now. Lmk wyt.